slave-base/contrib/settings.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There is a confusion between maven35 and slave-base.

Reply:
removed